src/NHSD.BuyingCatalogue.Ordering.Domain/ServiceRecipient.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace NHSD.BuyingCatalogue.Ordering.Domain
+{
+    public sealed class ServiceRecipient
+    {
+        [Key]

```

Comment:
Shouldn't need these attributes. Would you mind reviewing the use of the Entity Framework mapping configuration, should be an option here, which would free the domain object from persistence concerns?

Reply:
Added a key mapping to the application db context, however I had a problem with the foreign key attribute working